core/src/test/java/org/testcontainers/utility/MountableFileTest.java

```diff
@@ -81,6 +82,14 @@ public void forHostPathWithPlus() throws Exception {
         assertFalse("The resolved path does not contain an escaped space", mountableFile.getResolvedPath().contains(" "));
     }
 
+    @Test
+    public void forHostPathSpecialLocationDev() throws Exception {
+        final MountableFile mountableFile = MountableFile.forHostPath("/dev/shm");
+
+        final String resolvedPath = mountableFile.getResolvedPath();
+        assertThat("the /dev/shm path should always be direct", resolvedPath, startsWith("/dev/shm"));
```

Comment:
It would be nice if we could set this up so that it runs thinking it's on a Windows machine. Probably do-able.

However, we do have the safeguard that this test will be run on Windows CI at some point. Maybe that's enough?